packages/nimble-components/src/utilities/style/types.ts

```diff
@@ -0,0 +1,8 @@
+export const ZIndexLevels = {
+    justBelowDefault: '-1',
+    justAboveDefault: '1',
+    high: '100',
+    higher: '1000',
+    veryHigh: '10000'
```

Comment:
I'm struggling with the proposed names, because they don't quite convey an appropriate relative meaning to each other. For instance, `higher` could be thought to be "higher" than `veryHigh`. Ultimately, I think the goal of the enum is just to provide a mechanism so that we don't set a `z-index` to some arbitrary value. This means we could get _real_ specific with naming like `defaultPlus1`, `defaultPlus10`, etc... (maybe). I'm not sure if I have any better suggestions, but if we went with the current proposed scheme, I would think we want something where the largest value is called `highest`.